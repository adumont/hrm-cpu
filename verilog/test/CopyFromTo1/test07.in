56 a8
36 78
38 51
17 5d
41 be
26 59
41 9b
31 cc
22 99
33 a4
05 81
20 3f
33 63
53 62
57 dd
50 8c
07 b1
51 d8
07 c7
14 09
08 c9
57 25
51 f7
23 ad
42 df
03 b3
26 70
27 a7
37 de
46 b9
36 ec
39 9a
37 c2
52 08
31 04
59 6f
50 2f
12 cc
35 3b
16 b9
39 f1
29 69
16 f5
26 e9
08 3b
02 cf
58 81
37 1a
11 2f
