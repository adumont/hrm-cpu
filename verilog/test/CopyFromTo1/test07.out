a8
78
51
5d
be
59
9b
cc
99
a4
81
3f
63
62
dd
8c
b1
d8
c7
09
c9
25
f7
ad
df
b3
70
a7
de
b9
ec
9a
c2
08
04
6f
2f
cc
3b
b9
